src/CommandHandling/AuthorizedCommandBus.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace CultuurNet\UDB3\CommandHandling;
+
+use Broadway\CommandHandling\CommandBusInterface;
+use CultuurNet\UDB3\Offer\Commands\AuthorizableCommandInterface;
+use CultuurNet\UDB3\Security\CommandAuthorizationException;
+use CultuurNet\UDB3\Security\SecurityInterface;
+use CultuurNet\UDB3\Security\UserIdentificationInterface;
+
+class AuthorizedCommandBus extends CommandBusDecoratorBase implements AuthorizedCommandBusInterface
+{
+    /**
+     * @var UserIdentificationInterface
+     */
+    private $userIdentification;
+
+    /**
+     * @var SecurityInterface
+     */
+    private $security;
+
+    /**
+     * AuthorizedCommandBus constructor.
+     * @param CommandBusInterface $decoratee
+     * @param UserIdentificationInterface $userIdentification
+     * @param SecurityInterface $security
+     */
+    public function __construct(
+        CommandBusInterface $decoratee,
+        UserIdentificationInterface $userIdentification,
+        SecurityInterface $security
+    ) {
+        parent::__construct($decoratee);
+
+        $this->userIdentification = $userIdentification;
+        $this->security = $security;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function dispatch($command)
+    {
+        $authorized = true;
```

Comment:
Luc, ik begrijp waarom je dit doet, om te zorgen dat authorized op true staat indien een command geen authorizable command is. Maar ik vind dat gevaarlijk. Moest er iets mislopen staat het sowieso op true. Ik ben eerder van het pessimistische soort ivm security. Op false zetten, indien authorizable geef je authorizatie terug, indien geen authorizable command geef je true terug. Wat vind jij?